src/App.js

```diff
@@ -0,0 +1,48 @@
+import './App.css';
+import React, { useEffect } from 'react';
+import Questions from './questions/Questions'
+import Images from './images/Images'
+
+function App() {
+
+  useEffect(() => {
+    const interval = setInterval(() => {
```

Comment:
Try to implement this with React, do not use the `document.getElementById` without an urgent need for React.

Reply:
using ref?

Reply:
> Try to implement this with React, do not use the `document.getElementById` without an urgent need for React.

using ref?